### ISSUE TITLE
invalid device symbol

### ISSUE THREAD

**Issue by kingFzh** [niessner/BundleFusion#30]
Source\Solver\CUDASolverBuilding.cpp(135):cutilCheckMsg() CUTIL CUDA error:CUDASolverBuilding:
(13) invalid device symbol
![bundle](https://user-images.githubusercontent.com/44972305/48393783-e5df9a00-e74b-11e8-962d-31fc5bb82846.png)

**Comment by kingFzh** [comment]
![bundle](https://user-images.githubusercontent.com/44972305/48393464-a82e4180-e74a-11e8-89f4-55c98b0f4832.png)

**Comment by h1063135843** [comment]
I have met the same question as you. Have you solved it?

**Comment by kingFzh** [comment]
> 
> 
> I have met the same question as you. Have you solved it?

I haven't solved the problem yet.  I have no idea.
Do have the weChat? we can make a friend and talk about it.

**Comment by kingFzh** [comment]
> 
> 
> I have met the same question as you. Have you solved it?

**Comment by h1063135843** [comment]
Yeah,it seems that you should set the correct code generation which matches your graphic such as compute_61,sm_61. But the question turned to the problem mentioned in another issue, FriedLiver Stuck.

**Comment by kingFzh** [comment]
> FriedLiver Stuck
I have matched the graphic ,but the problem is the same as before and still haven't solved.

**Comment by h1063135843** [comment]
It;s a pity. The problem is sure about CUDA. Maybe you could ran it on another machine. I have the weChat.

**Comment by kingFzh** [comment]
> 
> 
> It;s a pity. The problem is sure about CUDA. Maybe you could ran it on another machine. I have the weChat.

What's your weChat?

**Comment by h1063135843** [comment]
![mmqrcode1541347512356](https://user-images.githubusercontent.com/26903569/48471339-a9d13580-e82e-11e8-8d0a-2d79b9b4aca3.png)

**Comment by BrigitteBai** [comment]
I met the same question，too. Have you solved it?

**Comment by Tomas-Lee** [comment]
I have solved this problem, that is the Graphic Card for display must be in accordance with the one for cuda Computing.

**Comment by aaaaangel** [comment]
I have solved this problem by setting -arch=sm_30 in CMakeLists.txt in [https://github.com/nonlinear1/BundleFusion_Ubuntu_V0](url).
